src/module.c

```diff
@@ -105,9 +105,10 @@ int TSDB_info(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     CompactionRule *rule = series->rules;
     int ruleCount = 0;
     while (rule != NULL) {
-        RedisModule_ReplyWithArray(ctx, 3);
+        RedisModule_ReplyWithArray(ctx, 4);
         RedisModule_ReplyWithString(ctx, rule->destKey);
         RedisModule_ReplyWithLongLong(ctx, rule->bucketDuration);
+        RedisModule_ReplyWithLongLong(ctx, rule->timestampAlignment);
```

Comment:
putting a new info field in the middle might break some clients that parse this based on order...